tools/dockerfile/grpc_artifact_python_manylinux2014_aarch64/Dockerfile

```diff
@@ -0,0 +1,31 @@
+# Copyright 2020 The gRPC Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# The aarch64 wheels are being crosscompiled to allow running the build
+# on x64 machine. The dockcross/manylinux2014-aarch64 image is a x86_64
+# image with crosscompilation toolchain installed.
+# Use an older version of dockcross image that has gcc4.9.4 because it was built
+# before https://github.com/dockcross/dockcross/pull/449
+FROM dockcross/manylinux2014-aarch64:20200929-608e6ac
+
+# Update the package manager
+RUN yum update -y && yum install -y curl-devel expat-devel gettext-devel openssl-devel zlib-devel
```

Comment:
Are these installing the aarch64 dev libs? When I tried it, it pulled in the x86 version.

Reply:
nope, this is installing x86 libs. I guess the reason why this doesn't matter is that they are actually not strictly required for a successful build (I copied the line from x64 dockerfile and didn't investigate whether they are really needed).